blocks/quote/quote.css

```diff
@@ -47,3 +47,28 @@
   position: relative;
   font-style: normal;
 }
+
+.block.quote.highlight {
```

Comment:
@JiangLong2019 I think this is a use case of text block.
For text block we have one light background variant.
You can implement left border variant in that.
And for text "IMPORTANT" we can check with Dejan if its necessary to be in orange color

Reply:
@jindaliiita You mean "Text Block with light background", let me check

Reply:
Yes

Reply:
We can simply add one more variant to text block i.e. left border